terraform/paas/terraform_qa.tfvars

```diff
@@ -0,0 +1,16 @@
+app = {
+  name         = "qa-publish-teacher-training-courses"
+  docker_image = "dfedigital/publish-teacher-training:latest"
+  hostname     = "qa-publish-teacher-training-courses"
+  space        = "find-qa"
```

Comment:
Should this be `publish-qa`?

Reply:
Colin asked me to use the same find-qa space for all the apps.